static/js/topic_page/types.tsx

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
were we going to do away with these 'util' like modules? maybe this could be topic_metadata?